app/helpers/course/assessment/submission/submissions_autograded_helper.rb

```diff
@@ -31,7 +31,8 @@ def nav_class(step)
     return 'completed' if step <= max_step
   end
 
+  # Fallback to the last answer if current_answer flag is not set on any answer.
   def current_answer
-    @answers.last
+    @answers.find { |ans| ans.current_answer == true } || @answers.last
```

Comment:
Is this still needed? the current_answer here is the answer of that step. different from the one that introduced in this PR.